cmd/osbuild-composer-dbjobqueue-tests/tern.conf

```diff
@@ -0,0 +1,7 @@
+[database]
+host = 127.0.0.1
+port = 5432
+database = osbuildcomposer
+user = postgres
+password = {{env "DBJOBQUEUE_TESTS_DB_PASS"}}
```

Comment:
Considering the password 'foobar' is in plaintext in the test itself, I think it's fine to just use 'foobar' here.